doc/mgr/dashboard.rst

```diff
@@ -65,31 +66,33 @@ aspects of your Ceph cluster:
   additional information and performance metrics gathered by the
   :ref:`mgr-prometheus`. See :ref:`dashboard-grafana` for details on how to
   configure this functionality.
-* **Cluster logs**: Display the latest updates to the cluster's event and audit
-  log files.
+* **Cluster logs**: Displays the latest updates to the cluster's event and
+  audit log files. Log entries can be filtered by priority, date or keyword.
 * **Hosts**: Provides a list of all hosts associated to the cluster, which
   services are running and which version of Ceph is installed.
 * **Performance counters**: Displays detailed service-specific statistics for
   each running service.
 * **Monitors**: Lists all MONs, their quorum status, open sessions.
-* **Configuration Editor**: View all available configuration options,
+* **Configuration Editor**: Displays all available configuration options,
   their description, type and default values and edit the current values.
-* **Pools**: List all Ceph pools and their details (e.g. applications, placement
-  groups, replication size, EC profile, CRUSH ruleset, etc.)
+* **Pools**: Lists all Ceph pools and their details (e.g. applications,
+  placement groups, replication size, EC profile, CRUSH ruleset, etc.)
 * **OSDs**: Lists all OSDs, their status and usage statistics as well as
-  detailed information like attributes (OSD map), metadata, performance counters
-  and usage histograms for read/write operations. Mark OSDs as up/down/out,
-  perform scrub operations. Select between different recovery profiles to adjust
-  the level of backfilling activity.
+  detailed information like attributes (OSD map), metadata, performance
+  counters and usage histograms for read/write operations. OSDs can be marked
+  as up/down/out, they can be purged and reweighted, scrub operations can be
```

Comment:
In proper English style, a list of sentences like this one needs to be separated by semicolons with an `and` before the final sentence. I.e.:

`OSDs can be marked up/down/out; OSDs can be purged and reweighted; and scrub operations can be performed.`

Reply:
looking at how the RBD section (below) is phrased, however, I would suggest to rephrase this whole section as follows:

"Mark OSDs up/down/out, purge and reweight OSDs, perform scrub operations, select different profiles to adjust the level of backfilling activity."